tracing/src/lib.rs

```diff
@@ -753,6 +756,7 @@ pub use self::{
 
 #[doc(inline)]
 pub use self::span::Span;
+#[cfg(feature = "attributes")]
```

Comment:
Can we add a `#[cfg_attr(docsrs, doc(cfg(feature = "attributes")))]` attribute as well? That way, the feature flag will be shown in the docs.